lib/index.js

```diff
@@ -1220,6 +1220,52 @@ module.exports = () => {
         return ret === 1;
       }
 
+      /**
+       * @summary add a tweak to a point
+       * @arg {Uint8Array} point - public key
+       * @arg {Uint8Array} tweak - scalar to add to the point
+       * @arg {boolean} compressed - whether the resulting point should be compressed
+       * @returns {Uint8Array} public key resulting from the point addition
+       */
+      function pointAddScalar(point, tweak, compressed) {
+        if (!compressed) compressed = true;
```

Comment:
I didn't get this. We force the flag to always be true? Whatìs its usefulness then?

Reply:
Right, this is an error. It should be true only if compressed is undefined. Added the fix + some assert statement in tests.